site/owid.entry.ts

```diff
@@ -67,6 +69,11 @@ window.runSiteFooterScripts = () => {
     }
 }
 
+Bugsnag.start({
+    apiKey: "0c88069ddc728aedd91001c376c0fe3c",
+    plugins: [new BugsnagPluginReact()],
+})
+
```

Comment:
Can this throw an error? Is it worth wrapping in try/catch?

Reply:
Also, let's take the API key from `.env`?

Reply:
Good call, thanks!